scripts/zkai_proof_pressure_scaling_claim_pack_gate.py

```
@@ -373,6 +379,51 @@ def validate_timing_payload(payload: dict[str, Any]) -> None:
+    for artifact_any in artifacts:
+        artifact = require_dict(artifact_any, "d128 four-head seq64 manifest artifact")
+        artifact_id = string_field(artifact.get("id"), "manifest artifact id")
+        if artifact_id not in expected or artifact_id in seen:
+            raise ProofPressureScalingClaimPackError("d128 four-head seq64 manifest artifact id drift")
+        seen.add(artifact_id)
+        expected_path, expected_size = expected[artifact_id]
+        if artifact.get("path") != expected_path or artifact.get("size_bytes") != expected_size:
+            raise ProofPressureScalingClaimPackError("d128 four-head seq64 manifest artifact metadata drift")
+        digest = string_field(artifact.get("sha256"), "manifest artifact sha256")
+        if len(digest) != 64 or any(ch not in "0123456789abcdef" for ch in digest):
+            raise ProofPressureScalingClaimPackError("d128 four-head seq64 manifest digest drift")
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)